runbotics/runbotics-scheduler/src/websocket/bot/bot.gateway.ts

```diff
@@ -124,4 +123,34 @@ export class BotWebSocketGateway implements OnGatewayDisconnect, OnGatewayConnec
         await this.botProcessEventService.updateProcessInstanceEvent(processInstanceEvent, bot);
         this.logger.log(`<= Success: process-instance-event (${processInstanceEvent.executionId}) updated by bot (${installationId}) | step: ${processInstanceEvent.step}, status: ${processInstanceEvent.status}`);
     }
+
+    private async changeEventStatusOnInstanceTermination(bot: IBot, processInstance: IProcessInstance, retryCount = 0) {
```

Comment:
avoid linking words like `on` in the names of functions or variables, there is a better way

Reply:
done